app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,17 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="http"
+                    android:host="community.exoplatform.com"
+                    android:pathPrefix="/portal" />
```

Comment:
We can remove this as the app won't connect to an app with clear text connection